WORKSPACE

```diff
@@ -6,6 +6,16 @@ grpc_deps()
 
 grpc_test_only_deps()
 
+# Initialize Google APIs with only C++ and Python targets
+load("@com_google_googleapis//:repository_rules.bzl", "switched_rules_by_language")
+
+switched_rules_by_language(
```

Comment:
@gnossen  is using `switched_rules_by_language` ok by you?

Reply:
Yes. Actually, I really like the pattern. I think it's something that we might want to try introducing to our own repo at some point.